.github/workflows/test_inf2.yml

```diff
@@ -38,20 +38,20 @@ jobs:
       - name: Run CLI tests
         run: |
           source aws_neuron_venv_pytorch/bin/activate
-          HF_TOKEN_OPTIMUM_NEURON_CI=${{ secrets.HF_TOKEN_OPTIMUM_NEURON_CI }} pytest -m is_inferentia_test tests/cli
+          HF_TOKEN=${{ secrets.HF_TOKEN_OPTIMUM_NEURON_CI }} pytest -m is_inferentia_test tests/cli
       - name: Run exporters tests
         run: |
           source aws_neuron_venv_pytorch/bin/activate
-          HF_TOKEN_OPTIMUM_NEURON_CI=${{ secrets.HF_TOKEN_OPTIMUM_NEURON_CI }} pytest -m is_inferentia_test tests/exporters
```

Comment:
Is this environment variable only used in the two workflows you changed?

Reply:
I saw Trainium tests are using it as well, do you want me to change trainium CIs as well?

Reply:
Let's do it in another PR.